star_ratings/models.py

```diff
@@ -13,15 +13,17 @@
 
 class AggregateRatingManager(models.Manager):
     def ratings_for_model(self, item):
+        if isinstance(item, AggregateRating):
+            raise TypeError("AggregateRating manager 'ratings_for_model' expects model to be rated, not AggregateRating model.")
         ct = ContentType.objects.get_for_model(item)
         aggregate, created = self.get_or_create(content_type=ct, object_id=item.pk)
         return aggregate
 
-    def rate(self, instance, score, user, ip=None):
-        if isinstance(instance, AggregateRating):
-            raise Exception('AggregateRating manager expects model to be rated, not AggregateRating model.')
-        ct = ContentType.objects.get_for_model(instance)
-        existing_rating = Rating.objects.for_instance_by_user(instance, user)
+    def rate(self, item, score, user, ip=None):
```

Comment:
I feel like `instance` is a good name here - what's the thinking behind changing it to `item`?